StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetFormFactory/PaymentSheetFormFactory+FormSpec.swift

```diff
@@ -7,38 +7,111 @@
 //
 
 import Foundation
+import StripeCore
 @_spi(STP) import StripeUICore
 
 extension PaymentSheetFormFactory {
-
     func specFromJSONProvider(provider: FormSpecProvider = FormSpecProvider.shared) -> FormSpec? {
         guard let paymentMethodType = PaymentSheet.PaymentMethodType.string(from: paymentMethod) else {
             return nil
         }
         return provider.formSpec(for: paymentMethodType)
     }
 
-    func makeFormElementFromSpec(spec: FormSpec) -> FormElement {
+    func makeFormElementFromSpec(
+        spec: FormSpec,
+        additionalElements: [Element] = []
+    ) -> PaymentMethodElementWrapper<FormElement> {
         let elements = makeFormElements(from: spec)
-        return FormElement(autoSectioningElements: elements, theme: theme)
+        let formElement = FormElement(
+            autoSectioningElements: elements + additionalElements,
+            theme: theme)
```

Comment:
SwiftLint doesn't correct this and it isn't a violation. SwiftFormat does enforce the latter style but we are not using it. I'm ambivalent on which style to use and I go with what feels better depending on the case, in this case for example, since it is only two lines I feel like keeping the parenthesis in the same line looks better.

Reply:
I'm pretty sure it's supposed to though? Look at the 2nd Triggering Example [here](https://realm.github.io/SwiftLint/multiline_arguments_brackets.html)

Reply:
hmm, you're right, but we don't have that rule enabled and I hadn't seen it, I'll enable it and run an autofix on the repo in a separate PR.